src/components/student/Transcript.js

```diff
@@ -1,18 +1,73 @@
 import React, {useState, useEffect} from 'react';
-
+import {SERVER_URL} from '../../Constants';
 // students gets a list of all courses taken and grades
 // use the URL /transcript?studentId=
-// the REST api returns a list of EnrollmentDTO objects 
-// the table should have columns for 
+// the REST api returns a list of EnrollmentDTO objects
+// the table should have columns for
 //  Year, Semester, CourseId, SectionId, Title, Credits, Grade
 
-const Transcript = (props) => {
-     
+const Transcript = () => {
+    const [transcriptData, setTranscriptData] = useState([]);
+    const [loading, setLoading] = useState(true);
+    const [error, setError] = useState(null);
+    const studentId = "3"
+
+
+        const fetchTranscriptData = async () => {
+        try {
+            const response = await fetch(`${SERVER_URL}/transcripts?studentId=3`);
+            if (!response.ok) {
+                throw new Error('Failed to fetch transcript data');
+            }
+            const data = await response.json();
+            setTranscriptData(data);
+            console.log(data.length);
+            setLoading(false);
+        } catch (error) {
+                setError(error.message);
+                setLoading(false);
+           }
+        };
+        useEffect(() => {
+        fetchTranscriptData();
+    }, []);
+
+
+
     return(
-        <> 
-            <h3>Not implemented</h3>
-        </>
+        <div>
+
+            <h3>Student Transcript</h3>
+            <table>
```

Comment:
Transcript is not centered on the page.  May want to add classname="Center" to this table tag to center it for uniformity.

Reply:
Added a CSS property to center